### ISSUE TITLE
I think notebook 5 probably needs a bit more explanation. From looking at the source code it seems clear that `modify_batch` and its sub-functions are doing a lot of heavy lifting, and from looking at the data folder (`data/cm2015_StateMod/StateMod/scenarios/S0_1`) I guess it is creating these files - but I feel this puts quite a lot on a user to figure out what's going on there.

### ISSUE THREAD

**Issue by crvernon** [IMMM-SFA/statemodify#52]

**Comment by barneydobson** [comment]
In addressing this - I think it would be a good opportunity to provide some evidence for this part of the paper:
_"If a user has access to a given model’s input and output files, many of the methods introduced in statemodify could be adapted for adjusting input files and compressing and visualizing the output files"_

You can go into some of the key functions in `statemodify` and explain how they work via the notebook, this will give people hoping to adapt it to another model the best chance of being able to do so.

**Comment by rg727** [comment]
We have added additional information on the `modify_batch( )` function in this notebook and made it clear that it is just running other `statemodify` functions simultaneously. In regards to the second comment, the format and structure of the StateMod input files is very specific and much additional functionality has been built in to address the unique formatting of the files that is difficult to explain concisely in a notebook. However, at a high level, users of `statemodify` can draw inspiration from the types of uncertainties sampled, the sampling methods, and the use of a Hidden Markov Model-based streamflow generator. At the end of each notebook, we have linked to the source code for each `statemodify` function as well.
